etna/lib/etna/clients/magma/workflows/crud_workflow.rb

```diff
@@ -11,7 +11,7 @@ def initialize(args)
         end
 
         def lookup_record(model_name, record_id)
-          if (cached = @cache.dig(model_name, record_id))
+          if (cached = (@cache ||= {}).dig(model_name, record_id))
```

Comment:
I was getting an error about `nil Class` for `@cache`, so maybe an issue with doing a lookup before an update?

Reply:
Ah, doh!  Yeah, makes sense.  I think I added the cache mechanism as a last minute optimization and missed this.